.github/workflows/ci.yml

```diff
@@ -228,7 +228,7 @@ jobs:
       run: cross check --all --target ${{ matrix.target }}
 
     - name: test
-      run: cross test --all --target ${{ matrix.target }} -- --test-threads=8
+      run: cross test --all --target ${{ matrix.target }} -- --test-threads=4
```

Comment:
why do we have to keep this change Oo?

Reply:
TLDR; cross + android makes memory usage grow like crazy. We're already seeing this on other branches too. We're right at the watermark here. Reducing parallelism on these gives more room. 

This is not the way to go about it as eventually we will hit `test-threads=1` and this grows runtime considerably which further clogs CI. I'll tackle this sooner rather than later.

Reply:
👍